py-polars/polars/interchange/column.py

```diff
@@ -148,32 +150,46 @@ def get_chunks(self, n_chunks: int | None = None) -> Iterator[PolarsColumn]:
 
     def get_buffers(self) -> ColumnBuffers:
         """Return a dictionary containing the underlying buffers."""
```

Comment:
I think this docstring must be adapted to clearly explain that for `String` and  `Binary` types we return a `buffer` that matches expectations of the DataFrame protocol.

Maybe this function should be private as well? Then we can change the outputs more easily if we have something better.

Reply:
This is the protocol itself, we are not allowed to change it. In the Series method `Series._get_buffers` I have included this information:

```
        Warnings
        --------
        The underlying buffers for `String` Series can not be represented in this
        format. Instead, the buffers are converted to a values and offsets buffer.
        This is used for the dataframe interchange protocol.
```

Reply:
Oh, right. I didn't read properly. As long as it is not public `get_buffer` on our core datastructures it is fine.